tests/Melodiia/Bridge/Symfony/EventListener/ExceptionListenerTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Biig\Melodiia\Test\Bridge\Symfony\EventListener;
+
+use Biig\Melodiia\Bridge\Symfony\EventListener\ExceptionListener;
+use Biig\Melodiia\MelodiiaConfigurationInterface;
+use PhpSpec\ObjectBehavior;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+
+class ExceptionListenerTest extends TestCase
+{
+    public function testItDoesNothingIfNotMelodiiaRoute()
+    {
+        /** @var MelodiiaConfigurationInterface|ObjectBehavior $config */
+        $config = $this->prophesize(MelodiiaConfigurationInterface::class);
+        $config->getApiEndpoints()->shouldBeCalled()->willReturn(['/api/v1/']);
+        $config = $config->reveal();
+
+        /** @var Request|ObjectBehavior $request */
+        $request = $this->prophesize(Request::class);
+        $request->getRequestUri()->shouldBeCalled()->willReturn('/random-url');
+        $request = $request->reveal();
+
+        $listener = new ExceptionListener($config, 'controller');
+        $event = $this->prophesize(GetResponseForExceptionEvent::class);
+        $event->getRequest()->willReturn($request);
+        $listener->onKernelException($event->reveal());
+    }
```

Comment:
IMO you should test the parent method `onKernelException` isn't called

Reply:
This class is already tested inside Symfony. the test is only here to be sure we return if it's not melodiia related.